core/src/main/scala/io/finch/request/package.scala

```diff
@@ -768,6 +800,8 @@ package object request {
         case Some(length) if length > 0 => Some(RequestBody(req)).toFuture
         case _ => Future.None
       }
+
+    override def toString: String = "body"
```

Comment:
Using `RequiredBody` actually returns a `ConstReader`. I didn't think that the `ConstReader`'s toString should return something about a request body.